_news/174.md

```diff
@@ -0,0 +1,142 @@
+All feedback is welcome, even if it is only to say that it works for you.
+We encourage users to try it out and give us feedback either by creating an issue at
+[https://github.com/erlang/otp/issues](https://github.com/erlang/otp/issues) or by
+posting to [Erlang Forums](https://erlangforums.com/).
+
+All artifacts for the release can be downloaded from the
+[Erlang/OTP Github](https://github.com/erlang/otp/releases/tag/OTP-28.0-rc1) release
+and you can view the new documentation at
+[https://erlang.org/documentation/doc-16.0-rc1/doc](https://erlang.org/documentation/doc-16.0-rc1/doc/).
+You can also install the latest release using [kerl](https://github.com/kerl/kerl) like this:
+
+```
+kerl build 28.0-rc1 28.0-rc1.
+```
+
+Starting with this release, a source [Software Bill of Materials
+(SBOM)](https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf)
+will describe the release on the [Github
+Releases](https://github.com/erlang/otp/releases) page. We welcome
+feedback on the SBOM.
+
+Erlang/OTP 28 is a new major release with new features, improvements
+as well as a few incompatibilities. Some of the new features are
+highlighted below.
+
+Many thanks to all contributors!
+
+
+# HIGHLIGHTS
+
+## New language features
+
+- Comprehensions have been extended with "zip generators" allowing
+  multiple generators to be run in parallel. For example,
+  `[A+B || A <- [1,2,3] && B <- [4,5,6]]` will produce `[5,7,9]`.
+
+- Generators in comprehensions can now be strict, meaning that if
+  the generator pattern does not match, an exception will be raised
+  instead of silently ignore the value that didn't match.
+
+- It is now possible to use any base for floating point numbers as per
+  [EEP 75: Based Floating Point Literals](https://www.erlang.org/eeps/eep-0075).
+
+
+## Compiler and JIT improvements
+
+- For certain types of errors, the compiler can now suggest
+  corrections.  For example, when attempting to use variable `A` that
+  is not defined but `A0` is, the compiler could emit the following
+  message: `variable 'A' is unbound, did you mean 'A0'?`
+
+- The size of an atom in the Erlang source code was limited to 255
+  bytes in previous releases, meaning that an atom containing only
+  emojis could contain only 63 emojis. While atoms are still only
+  allowed to contain 255 characters, the number of bytes is no longer
+  limited.
+
+- The `warn_deprecated_catch` option enables warnings for use of
+  old-style catch expressions on the form `catch Expr` instead of the
+  modern `try` ... `catch` ... `end`.
+
+- Provided that the map argument for a `maps:put/3` call is known to
+  the compiler to be a map, the compiler will replace such calls with
+  the corresponding update using the map syntax.
+
+- Some BIFs with side-effects (such as `binary_to_atom/1`) are
+  optimized in `try` ... `catch` in the same way as guard BIFs in
+  order to gain performance.
+
```

Comment:
Suggested addition:

 - Improved performance of the compiler's alias analysis pass to roughly halve the time required for the analysis. The improved performance allows improvements in the analysis to handle chains of extracts where it previously it only handled the innermost embed+extract pair and conservatively classified longer chains as producing an aliased value. (#8695)

Reply:
I added an edited version of your suggestion which is lighter on technical details and also pointing what the benefit is for the user:

>  * The compiler’s alias analysis pass is now both faster and less conservative, allowing optimizations of records and binary construction to be applied in more cases.

I've created a reminder for myself to add a release note for RC2 about compiler optimizations.

Reply:
I like what you did with my blurb!